esp/esp/users/models/__init__.py

```diff
@@ -1473,7 +1473,7 @@ class TeacherInfo(models.Model, CustomFormsLinkModel):
         ('shirt_type', 'Shirt type'),
```

Comment:
the "shirt size" and "shirt type" fields produce long answer fields, which seems wrong. They should either be short answers, or even radio buttons / dropdowns